security/advancedtls/advancedtls_test.go

```diff
@@ -679,3 +681,105 @@ func TestOptionsConfig(t *testing.T) {
 		})
 	}
 }
+
+const (
+	tlsEcdheRsaWith3desEdeCbcSha uint16 = 0xc012
```

Comment:
Why do we have these package-level variables? I think it might be better to have them in the test function?

Reply:
Ok! I'll move these inside the test function.